### PR TITLE
switch to blob storage for sound files

### LINKED ISSUE #66: Switch from file storage to blob storage

### LINKED ISSUE #72: Reduce size of bot image
Split into a builder / runtime image without make, python, etc.

### PR DESCRIPTION
**Pull request by UncleDave**
Fixes #66
Fixes #72

### CODE REVIEW

File: scripts/import-sounds-to-mongo.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 // Use this script to import sounds from a folder into a mongo database.
 // Doesn't check for or clear existing documents, so running it twice will produce duplicates.
 
 const connectionUri = 'mongodb://localhost:27017';
-const soundsDirectory = '../bot/sounds';
+const soundsDirectory = 'sounds';
 
 function splitFileName(name: string): SoundFile {
   const fullName = name.toLowerCase();
```
Comment (stuckinab0x) [inline scripts/import-sounds-to-mongo.ts:13]:
```suggestion
  const fullName = name;
```

You're not doing this in the blob stuff so any sound that had caps gets `404` when it goes to look for it.

**Review by stuckinab0x — CHANGES_REQUESTED** [review]
Still getting a CORS error for sound preview btw, can't remember if you said that should be fixed or not.

**Comment by UncleDave** [comment]
> Still getting a CORS error for sound preview btw, can't remember if you said that should be fixed or not.

I added the `configure-blob-storage.` script to fix that.

File: scripts/copy-sounds-to-blob-storage.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { createReadStream } from 'node:fs';
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import { FilesService } from 'botman-sounds/src/files-service';
+
+const soundsDirectory = 'sounds';
```
[the remark this answers is not in the document]
Comment (UncleDave) [reply scripts/copy-sounds-to-blob-storage.ts:6]: It's relative to the directory you're running the script from, and I usually run it from the root of the repo with `ts-node scripts/whatever.ts`, but I could just use `__dirname` and then everyone is happy.

File: scripts/import-sounds-to-mongo.ts (hunk 1 above)
Comment (UncleDave) [reply scripts/import-sounds-to-mongo.ts:13]: Come to think of it I should probably just combine them into one "import" script now.

File: scripts/copy-sounds-to-blob-storage.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (stuckinab0x) [reply scripts/copy-sounds-to-blob-storage.ts:6]: oh yeah okay I don't really care then

**Comment by stuckinab0x** [comment]
> > Still getting a CORS error for sound preview btw, can't remember if you said that should be fixed or not.
> 
> I added the `configure-blob-storage.` script to fix that.

I did run it, dunno if I'm missing something that would make it not work.

**Comment by UncleDave** [comment]
> > > Still getting a CORS error for sound preview btw, can't remember if you said that should be fixed or not.
> > 
> > 
> > I added the `configure-blob-storage.` script to fix that.
> 
> I did run it, dunno if I'm missing something that would make it not work.

I brokeded it by adding `https`

**Review by stuckinab0x — APPROVED** [review]

**Pull request merged**